src/style.css

```diff
@@ -0,0 +1,48 @@
+body,
+li,
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+.li {
+  list-style: none;
+}
+
+.stars__star {
+  display: inline-block;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center left;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
```

Comment:
```suggestion

.stars {
  display: flex;
  justify-content: center;
}

.stars__star {
  display: inline-block;
  width: 16px;
  height: 16px;
  background-image: url(images/star.svg);
  background-repeat: no-repeat;
  background-position: center left;
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}

```

Reply:
Let's write classes for one element in one place, it's increasing readability of code